config/_default/menus/menus.en.yaml

```
@@ -967,9 +967,9 @@ main:
     parent: tracing_setup
     weight: 313
   - name: Version Tracking
```

better change the name, too
```suggestion
  - name: Deployment Tracking
```